snoop/data/digests.py

```diff
@@ -38,15 +41,25 @@ def gather(blob, collection_pk, **depends_on):
 
     tika_rmeta_blob = depends_on.get('tika_rmeta')
     if tika_rmeta_blob:
-        with tika_rmeta_blob.open(encoding='utf8') as f:
-            tika_rmeta = json.load(f)
-        rv['text'] = tika_rmeta[0].get('X-TIKA:content', "")
+        if isinstance(tika_rmeta_blob, ShaormaBroken):
+            rv['broken'].append(tika_rmeta_blob.reason)
+            log.warn("tika_rmeta task is broken; skipping")
```

Comment:
This means a lot of log traffic. Don't think it's of any use, you should see the `broken` list in the digest, right?

Reply:
Right. How about I change those to `log.debug`?

Reply:
sure

Reply:
Merge it afterwards